### ISSUE TITLE
EF Classic?

### ISSUE THREAD

**Issue by kanadaj** [mrahhal/Migrator.EF6#66]
Hey there, do you think there is any reasonable way to make this work with ZProject.EntityFramework.Classic? It's an EF6 fork that runs on .NET Core/netstandard, but has no migration tooling

**Comment by mrahhal** [comment]
Hey. This project depends on the `EntityFramework` package's migration tooling for the real work. The question here is whether ZProject.EntityFramework.Classic contains the equivalent migration types I'm using. I haven't used it so I don't really know. All in all, it's possible as I don't really depend on full .NET apart from using EF6 itself, but it'll possibly need lots of work.

**Comment by kanadaj** [comment]
That's a likely no. I think the `EntityFramework` package's tooling might only work with the legacy project format which is already a problem :(

**Comment by mrahhal** [comment]
Yeah, that's unfortunate. It was already a challenge to make this work with the fully available tooling in EF6.

**Comment by kanadaj** [comment]
It's honestly such a pain. I'm on a deadline to get an ASP.NET Core EF6 app moved from Azure to Linux VMs which would ideally include moving to .NET Core fully, and finally we have EF6 versions that run on Core but no migration tooling which is super important :(

**Comment by mrahhal** [comment]
We also had to make our peace with sticking with full .NET for one of our large projects because we need EF6. Nothing can be done here but wait for EF6 to become .NET standard.

**Comment by kanadaj** [comment]
Sadly, I cannot wait unless I am willing to risk running this web app on Mono, and I'm not sure that's a safe bet...
